base/cpu/openmpi4.1.0-ubuntu22.04/Dockerfile

```diff
@@ -108,4 +108,3 @@ RUN apt-get update && \
     apt-get update && \
     ACCEPT_EULA=Y apt-get install -y msodbcsql17
 
-

```

Comment:
This is not the right dockerfile. Are you making these changes manually or copy from ADO source?

Reply:
I'm making those changes manually